docs-chef-io/content/inspec/troubleshooting.md

```diff
@@ -22,3 +22,9 @@ See the relevant resource pack readme for instructions:
 - [inspec-aws README](https://github.com/inspec/inspec-aws#use-the-resources)
 - [inspec-azure README](https://github.com/inspec/inspec-azure#use-the-resources)
 - [inspec-gcp README](https://github.com/inspec/inspec-gcp#use-the-resources)
+
+## Error 173
```

Comment:
Might be clearer to say "Exit Code 173" - we don't have numbered error messages, and this somewhat implies that we do.

Reply:
It is exit code 174 when license is not set for inspec
Exit code 173 is when license is not entitled to inspec software